airflow/providers/amazon/CHANGELOG.rst

```diff
@@ -19,6 +19,56 @@
 Changelog
 ---------
 
+2.6.0
+.....
+
+Features
+~~~~~~~~
+
+* ``Add aws_conn_id to DynamoDBToS3Operator (#20363)``
+* ``Add RedshiftResumeClusterOperator and RedshiftPauseClusterOperator (#19665)``
+* ``Added function in AWSAthenaHook to get s3 output query results file URI  (#20124)``
+* ``Add sensor for AWS Batch (#19850) (#19885)``
+* ``Add state details to EMR container failure reason (#19579)``
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Fix backwards compatibility issue in AWS provider's _get_credentials (#20463)``
+* ``Fix deprecation messages after splitting redshift modules (#20366)``
+* ``ECSOperator: fix KeyError on missing exitCode (#20264)``
+* ``Bug fix in AWS glue operator when specifying the WorkerType &amp; NumberOfWorkers (#19787)``
```

Comment:
Should it have `&amp;` ?

Reply:
Not really :). But I can use the #20514 to fix it.

Reply:
Fixed it permanently (missing "| safe" in Jinja filter.